### ISSUE TITLE
syntax error please check

### ISSUE THREAD

**Issue by LuciferMrngSta** [updatehacks/ipcs-update-new-cameras-add-#2]
ipcs-update-new-cameras-add-# python3 scan.py
  File "scan.py", line 48
    print ""
           ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("")?

**Comment by updatehacks** [comment]
Ok, i'm fix this

вт, 22 окт. 2019 г., 21:40 LuciferMrngSta <notifications@github.com>:

**Comment by updatehacks** [comment]
Yes, i'm fix this, few moment ago! Please delete a my repository, and copy "ipcs-new.....".
